README.md

```diff
@@ -9,34 +9,37 @@ Get the main repository:
 git clone https://github.com/mean00/avidemux2.git
 ```
 
-Get submodules:
+It needs an additional i18n (translation files) submodule. You can get it with:
 ```
 git submodule update --init --recursive
 ```
 
-## Debian / Ubuntu and derivatives
-
-Required packages:
-```
-sudo apt install gcc build-essential cmake pkg-config yasm libsqlite3-dev  qtbase5-dev qttools5-dev-tools wget
-```
-
-Recommended packages:
+The application can be built with the following convenience script (it will check and install all dependencies):
 ```
-sudo apt install libxv-dev libvdpau-dev libva-dev libasound2-dev libpulse-dev libaom-dev libx264-dev libx265-dev libxvidcore-dev libvpx-dev libmad0-dev libmp3lame-dev libtwolame-dev libopus-dev libvorbis-dev libogg-dev libass-dev libaften-dev squashfs-tools
+bash makeAppImageBusterMinimal.sh
 ```
 
-Build with one of the provided scripts, e.g.:
-```
-bash makeAppImage.sh
-```
 
-After the project has been compiled, it can be run:
-```
-bash run_avidemux_template.sh
-```
+> ### Troubleshooting
+> In case not all the packages can be installed (unmet dependencies, broken packages, etc.), you can try to manually install packages.
+> Required packages:
+> ```
+> sudo apt install gcc build-essential cmake pkg-config yasm libsqlite3-dev  qtbase5-dev qttools5-dev-tools wget
+> ```
+> Recommended packages:
+> ```
+> sudo apt install libxv-dev libvdpau-dev libva-dev libasound2-dev libpulse-dev libaom-dev libx264-dev libx265-dev libxvidcore-dev libvpx-dev libmad0-dev libmp3lame-dev libtwolame-dev libopus-dev libvorbis-dev libogg-dev libass-dev libaften-dev squashfs-tools
+> ```
+> In case of missing non-essential package(s), try build with:
+> ```
+> bash makeAppImage.sh
```

Comment:
Have you verified that this works, especially with the declared set of essential packages? And if yes, which Debian (and eventually Ubuntu) versions are covered?

(IMHO, writing documentation is a hell, much more difficult and demanding than coding, especially when targeting audience without skills.)

Reply:
I did :)
Recently installed new system (Linux Mint 20.1), and the listed required packages are needed to be compiled.
This was also the case on my previous system.
Besides that, i always used makeAppImage.sh.

Reply:
Great, what package did provide AppImageAssistant, called by makeAppImage.sh?